src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -275,7 +284,40 @@ public HttpRequestActionBuilder searchForDrug(String searchQuery) {
 		return http("Search for Drug")
 				.get("/openmrs/ws/rest/v1/drug?name=" + searchQuery + "&v=" + customRepresentation);
 	}
-	
+	public HttpRequestActionBuilder saveCondition(String patientUuid){
+		String currentDate = ZonedDateTime.now().format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+
+		Map<String, Object> condition = new HashMap<>();
+		condition.put("clinicalStatus", Map.of(
+				"coding", new Object[]{Map.of(
+						"system", "http://terminology.hl7.org/CodeSystem/condition-clinical",
+						"code", "active"
+				)}
+		));
+		condition.put("code", Map.of(
+				"coding", new Object[]{Map.of(
+						"code", CONDITION_UUID,
+						"display", CONDITION_NAME
```

Comment:
It's okay to hard code it here.
```suggestion
						"display", "Back Pain"
```